CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Don't override paths in pyproject.toml with empty CLI paths (bcbnz, #228).
 * Run continuous integration tests for Python 3.9 (ju-sh, #232).
 * Use pathlib internally (ju-sh, #226).
+* Add [pre-commit](https://pre-commit.com) hook (Clément Robert #244).
```

Comment:
Can you merge this with the newest master, please? No need to rebase, I'll squash everything into one commit anyway.

Reply:
done !